pkg/config/default.go

```diff
@@ -669,12 +668,6 @@ func getDefaultSSHConfig() string {
 	return filepath.Join(dirname, ".ssh", "config")
 }
 
-func useUserConfigLocations() bool {
-	// NOTE: For now we want Windows to use system locations.
-	// GetRootlessUID == -1 on Windows, so exclude negative range
```

Comment:
Is this comment removed intentionally?

Reply:
I will add it back to `defaultTmpDir()`

Reply:
It will remain non-obvious in `defaultConfig`. Consider either adding the comment there as well, or keeping the existing `useUserConfigLocations` around, with the comment in there.